docs/MIG_calibration_error/MIG.md

```diff
@@ -0,0 +1,32 @@
+---
+description: MIG "init_calib_complete" signal does not go high during simulation
+date: 2020-10-21
+---
+
+# MIG "init_calib_complete" signal does not go high
+
+For a project, which utilizes 7-Series, I should use PL memory and **MIG** is the best IP choice for this purpose. In many websites, including [Xilinx documents](https://www.xilinx.com/support/documentation/ip_documentation/mig_7series/v4_1/ug586_7Series_MIS.pdf), you can find how to configure MIG IP. Even if there are many configuration parameters in MIG IP, if you know where to look, it is so easy to build it.
+
+After configuring your 7-Series MIG, you will notice that there is an important signal called "**init_calib_complete**". Calibration is the first stage of MIG if you run the code. MIG arranges phasors, clocks, etc and small write-read test with the help of 200 MHz reference clock in calibration stages, which detailed information can be found via [this link](https://www.xilinx.com/support/answers/51954.html). 
+
+However, if you debug your code,regardless of whether you choose "**Debug Signals for Memory Controller**" Mode On or not, and if init_calib_complete signal is not high, you cannot reach your PL memory (Even your PS memory), so AXI transaction error can show up. Therefore, there are some checkpoints that you should be sure about. I have listed step by step what you should check to correct calibration stage and use your PL memory:
+
+**1)** **Check MIG Parameters** --> As stated before, in MIG, there are tons of parameters especially PL memory port configuration, which can confuse a person so much. Therefore, if you have a problem with "**init_calib_complete**" signal, you should check your MIG configuration. I have listed the most important parameters and stages that you should be careful about below:
+
+Page Name    | Setting Name | Instruction
```

Comment:
Tablo karakterlerinin etrafındaki whitespace'leri kaldıralım. Pin Selection for Controller'ın ikinci sütunundaki ---- lerin uzun olmasının bir anlamı var mı?